ciphers/password_encryption.py

```diff
@@ -0,0 +1,22 @@
+from passlib.context import CryptContext 
+#importing cryptcontext from passlib
+
+pwd_context = CryptContext(
+        schemes=["pbkdf2_sha256"],
+        default="pbkdf2_sha256",
+        pbkdf2_sha256__default_rounds=30000
+)
+
+#A round is a part of the algorithm that runs many times in order to reduce "crackability".
+
+def encrypt_password(password):
+    return pwd_context.encrypt(password)
+
+
+def check_encrypted_password(password, hashed):
+    return pwd_context.verify(password, hashed)
+
+password_val=input("Enter the password :")
```

Comment:
This line will hang our automated testing as discussed in CONTRIBUTING.md.

This looks more like a how-to-use demonstration than an algorithm as discussed in CONTRIBUTING.md.